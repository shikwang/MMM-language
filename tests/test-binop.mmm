func int main() {
    
    print(3+6);
    print(3-6);
    print(3*6);
    print(6/3);
    return 0;
}