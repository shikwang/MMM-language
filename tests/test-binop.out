9
-3
18
2